README.md

```diff
@@ -15,14 +15,12 @@ To learn how to contribute to the MetaMask project itself, visit our [Internal D
     - If you are using [nvm](https://github.com/creationix/nvm#installation) (recommended) running `nvm use` will automatically choose the right node version for you.
     - If you install Node.js manually, ensure you're using npm@6
         - Install npm@6 using `npm install -g npm@6`
-- Install dependencies: `npm install`
+- Install dependencies: `npm ci`
     - If you have issues with node-sass compilation, try `npm rebuild node-sass`
-- Install gulp globally with `npm install -g gulp-cli`.
-- Build the project to the `./dist/` folder with `gulp build`.
-- Optionally, to rebuild on file changes, run `gulp dev`.
-- To package .zip files for distribution, run `gulp zip`, or run the full build & zip with `gulp dist`.
+- Build the project to the `./dist/` folder with `npm run dist`.
+- Optionally, to rebuild on file changes, run `npm start`.
```

Comment:
It might also be worth mentioning that this is a dev build, as opposed to `npm run dist` which is not.

Reply:
Good point, I'll update this line